src/HotChocolate/AspNetCore/src/AspNetCore/HttpPostMiddlewareBase.cs

```diff
@@ -211,7 +211,7 @@ protected async Task HandleRequestAsync(HttpContext context)
                 // we need to execute a request batch where we need to execute multiple
                 // fully specified GraphQL requests at once.
                 default:
-                    if (context.GetGraphQLServerOptions()?.EnableGetRequests ?? false)
+                    if (context.GetGraphQLServerOptions()?.EnableBatching ?? false)
```

Comment:
`EnableBatching` previously only worked, if the user didn't specify any `GraphQLServerOptions`, because then you would hit the default value `?? false` here. I don't really like that we specify the default for these flags in multiple places. We should probably make it so that `GraphQLServerOptions` can never be null and always is an object that contains the default values, unless overridden by the user.

Reply:
Its not ideal ... lets have a look at the middleware options for version 14 ... I have a couple of things i would like to refine,

Reply:
btw ... good catch :) must have been asleep.